ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -301,9 +301,11 @@ discontinuous_heaviside(const double& phi)
  * Eigen types have special alignment requirements and require a specific
  * memory allocator. This is a convenience type alias for a
  * <code>std::vector</code> with the correct allocator.
+ *
+ * @deprecated This alias is no longer necessary.
  */
 template <typename T>
-using EigenAlignedVector = std::vector<T, Eigen::aligned_allocator<T> >;
+using EigenAlignedVector = std::vector<T>;
```

Comment:
Don't we require `c++14` now? Should we use the deprecated attribute here?

Reply:
Nice catch (we require C++17 so we can add a deprecation message too)

Reply:
Fixed!